tests/text_content_detection.rs

```diff
@@ -55,70 +55,64 @@ async fn no_detections() -> Result<(), anyhow::Error> {
-                results: vec![
-                    Token {
-                        start: 0,
-                        end: 40,
-                        text: "This sentence does not have a detection.".into(),
-                    },
-                    Token {
-                        start: 41,
-                        end: 64,
-                        text: "Neither does this one.".into(),
-                    },
-                ],
-                token_count: 0,
-            }),
-        ),
-    );
+            });
+        then.pb(TokenizationResults {
+            results: vec![
+                Token {
+                    start: 0,
+                    end: 40,
+                    text: "This sentence does not have a detection.".into(),
+                },
+                Token {
+                    start: 41,
+                    end: 64,
+                    text: "Neither does this one.".into(),
+                },
+            ],
+            token_count: 0,
+        });
+    });
 
     let mut sentence_detector_mocks = MockSet::new();
-    sentence_detector_mocks.insert(
-        MockPath::post(TEXT_CONTENTS_DETECTOR_ENDPOINT),
-        Mock::new(
-            MockRequest::json(ContentAnalysisRequest {
+    sentence_detector_mocks.mock(|when, then| {
+        when.post()
+            .path(TEXT_CONTENTS_DETECTOR_ENDPOINT)
+            .json(ContentAnalysisRequest {
                 contents: vec![
                     "This sentence does not have a detection.".into(),
                     "Neither does this one.".into(),
                 ],
                 detector_params: DetectorParams::new(),
-            }),
-            MockResponse::json(vec![
-                Vec::<ContentAnalysisResponse>::new(),
-                Vec::<ContentAnalysisResponse>::new(),
-            ]),
-        ),
-    );
+            });
+        then.json(vec![
+            Vec::<ContentAnalysisResponse>::new(),
+            Vec::<ContentAnalysisResponse>::new(),
+        ]);
+    });
 
     let mut whole_doc_detector_mocks = MockSet::new();
-    whole_doc_detector_mocks.insert(
-        MockPath::post(TEXT_CONTENTS_DETECTOR_ENDPOINT),
-        Mock::new(
-            MockRequest::json(ContentAnalysisRequest {
+    whole_doc_detector_mocks.mock(|when, then| {
+        when.post()
+            .path(TEXT_CONTENTS_DETECTOR_ENDPOINT)
+            .json(ContentAnalysisRequest {
                 contents: vec!["This sentence has no detections.".into()],
                 detector_params: DetectorParams::new(),
-            }),
-            MockResponse::json(vec![Vec::<ContentAnalysisResponse>::new()]),
-        ),
-    );
+            });
+        then.json(vec![Vec::<ContentAnalysisResponse>::new()]);
+    });
 
     // Start orchestrator server and its dependencies
-    let mock_chunker_server = GrpcMockServer::new(chunker_id, chunker_mocks)?;
+    let mock_chunker_server = MockServer::new(chunker_id).with_mocks(chunker_mocks);
```

Comment:
Missing `.grpc()` option
```suggestion
    let mock_chunker_server = MockServer::new(chunker_id).grpc().with_mocks(chunker_mocks);
```